### PR TITLE
ZH-trans: add kubeadm-join-phase.md

### PR DESCRIPTION
**Pull request by xichengliudui**
source file: https://github.com/kubernetes/website/blob/release-1.14/content/en/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<!--
+`kubeadm join phase` is consistent with the [kubeadm join workflow](/docs/reference/setup-tools/kubeadm/kubeadm-join/#join-workflow),
+and behind the scene both use the same code.
+-->
+`kubeadm join phase` 与 [kubeadm join 工作](/docs/reference/setup-tools/kubeadm/kubeadm-join/#join-workflow)是一致的，而在底层，两者使用相同的代码。
+
+## kubeadm join phase {#cmd-join-phase}
+
+{{< tabs name="tab-phase" >}}
+{{< tab name="phase" include="generated/kubeadm_join_phase.md" />}}
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md]: looks like none of the referenced files exist in the 1.14 branch

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+title: kubeadm join phase
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md]: kubeadm join 阶段

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md]: 👍

**Comment by tengqm** [comment]
@xichengliudui Still working on this? Or should we close it for now?

**Pull request closed without merging**